### ISSUE TITLE
Freeze when reordering mouse profiles by dragging

### ISSUE THREAD

**Issue by ersi-dnd** [OtterBrowser/otter-browser#1452]
The procedure:

- Create at least two mouse profiles in Preferences/Advanced/Mouse
- Try to reorder them by dragging with the mouse
Result: Freeze

Interestingly, when force-closed in this state, Otter does not display the startup dialog at next start.

**Comment by Emdek** [comment]
@ersi-dnd, cannot reproduce.
Is there maybe some extra information that could help in reproducing it?

**Comment by ersi-dnd** [comment]
You are right. It's not a freeze. But the problem is there: When I try to reorder mouse profiles, I lose access to Otter window. 

However, I discovered that I can get out of the impasse by hitting Esc and then I regain access to Otter. This is regardless of the computer I use and it's there with clean profile also.

**Comment by Diablopc** [comment]
@ersi-dnd
I have this problem not only with the otter, but with all the Qt-based software when using the wacom tablet. It is especially easy to reproduce this - using the kvantum styling engine.
But I did not notice this problem with the mouse.

**Comment by Emdek** [comment]
@ersi-dnd, I've managed to reproduce it using 5.10 under Ubuntu, it seems to be something 5.10 specific, maybe updating to final 5.10 will be enough.
